electrical/calibration.MD

```diff
@@ -0,0 +1,183 @@
+<!-- Open Source Rover: Calibration
+Authors: Michael Cox, Eric Junkins, Olivia Lofaro
+Reference herein to any specific commercial product, process, or service by trade name, trademark, manu- facturer, or otherwise, does not constitute or imply its endorsement by the United States Government or the Jet Propulsion Laboratory, California Institute of Technology. ⃝c 2018 California Institute of Technology. Government sponsorship acknowledged.
+ 1
+-->
+# Open Source Rover: RoboClaw Calibration
+
+# 1. Calibration
+In order for the motors to all work properly and drive the rover correctly, we must calibrate the motors and encoders. We will use a program called Basic Micro Studio to assist with this. BasicMicro Motion Studio can be downloaded at
+
+* [Motion Studio Download](http://www.basicmicro.com/downloads)
+
+We will now go through and check the encoders and motors one at a time and perform the calibration process on each of them. First, make sure that either the robot is fully hanging such that every drive wheel is not in contact with the ground, or that you elevate each drive wheel as you attempt to spin that wheel.
+
+## 1.1 RoboClaw Setup
+1. Connect your computer (with Basic Micro Studio installed) to the RoboClaw via USB- micro USB cable
```

Comment:
would be good for these numbers to use N.N.N formatting

Reply:
done